src/Bicep.Core.Samples/Files/InvalidTypeDeclarations_LF/main.diagnostics.bicep

```diff
@@ -79,7 +79,6 @@ type objectWithInvalidPropertyDecorators = {
   fooProp: string
 
   @secure()
-//@[03:09) [BCP297 (Error)] Function "secure" cannot be used as a type decorator. (CodeDescription: none) |secure|
```

Comment:
Why did this error go away?

Reply:
The `@secure` decorator has been blocked on `type` statements and property declarations because the linter would not warn when a value containing a secret was used in an output. (The ARM engine has already been updated to redact any outputs containing nested secrets, which would have caused an odd user experience.)